internal/client/hub/hub.go

```diff
@@ -905,6 +906,19 @@ func getExternalApi(displayName string, description string,
 		e.Endpoints = endpoints
 	}
 
+	a, err := getAttributeAllowedlue(attribute, 0)
```

Comment:
did you mean to called the method "getAttributeAllowedValues" ?

Reply:
Yes, fixing it.